### ISSUE TITLE
switch off installLatestAwsSdk by default

### ISSUE THREAD

**Issue by dmarinuswoodwing** [projen/projen#2918]
As fixed in CDK here:

https://github.com/aws/aws-cdk/pull/23591

projen should also initialize `cdk.json` with `installLatestAwsSdk` set to `false` by default.

This can be done by adding:

```ts
  context: {
    '@aws-cdk/customresources:installLatestAwsSdkDefault': false,
  },
```

To projenrc.

Currently `npx projen new awscdk-app-ts` doesn't include these lines.

**Comment by mrgrain** [comment]
You're right! projen should probably follow all recommended feature flag values on generation.
